### PR TITLE
RSVP bugs fixed

### PR DESCRIPTION
**Pull request by MadhushaPrasad**
- Remove Lipsum text in login page
- Align Footer's Useful Links properly
- Change 'T' in team to 't'. ( Fill up the form and our team ... )
- Change Sign up/Login button in nav bar to match the heading in Sign up/Login page. (Ex: if login page heading in 'Login' change button text to 'Login' as well)
- Have a lighter color for footer icons

### CODE REVIEW

**Comment by sithumonline** [comment]
@MadhushaPrasad pipeline is failed plz fix that.

**Comment by sithumonline** [comment]
@MadhushaPrasad plz provide screenshot of changers.

**Comment by sithumonline** [comment]
`yarn prettier --write .` plz reformat code base.

File: components/Login/LoginInfo.tsx [hunk 1]
```diff
@@ -16,10 +16,6 @@ const LoginInfo = (props: LoginInfoProps): JSX.Element => {
   if (props.login) {
     return (
       <div className={outerContainerClass}>
-        <p className={paragraphClass}>
```
Comment (nisalrenuja) [inline components/Login/LoginInfo.tsx:19]: Please add content instead of removing

File: components/Login/LoginInfo.tsx [hunk 2]
```diff
@@ -32,10 +28,6 @@ const LoginInfo = (props: LoginInfoProps): JSX.Element => {
   } else {
     return (
       <div className={outerContainerClass}>
-        <p className={paragraphClass}>
```
Comment (nisalrenuja) [inline components/Login/LoginInfo.tsx:35]: Please add content Please add content instead of removing

**Review by nisalrenuja — CHANGES_REQUESTED** [review]

**Comment by MadhushaPrasad** [comment]
Screenshot 2022-06-06 173216
Screenshot 2022-06-06 173316
Screenshot 2022-06-06 174446

**Comment by MadhushaPrasad** [comment]
Screenshot 2022-06-07 114623

**Review by nisalrenuja — APPROVED** [review]

**Pull request merged**